features-json/webgl2.json

```diff
@@ -91,7 +91,7 @@
       "48":"n d #1 #5",
       "49":"n d #1 #5",
       "50":"n d #1 #5",
-      "51":"y",
+      "51":"n d #1 #5",
```

Comment:
Firefox Release Notes website claims that support for WebGL 2 was added in Firefox 51 (not in 52). Also other sources like e.g. [MDN's WebGL 2 browser compatibility data](https://developer.mozilla.org/en-US/docs/Web/API/WebGL2RenderingContext) and [WebGL article on Wikipedia](https://en.wikipedia.org/wiki/WebGL) claims the same.